server/routes/shared_api/autocomplete/helpers.py

```diff
@@ -181,6 +259,29 @@ def predict(queries: List[str], lang: str) -> List[ScoredPrediction]:
   return responses
 
 
+def fetch_place_id_to_dcid(
+    prediction_responses: List[ScoredPrediction]) -> Dict:
```

Comment:
Can the response type be more precise? (is `Dict[str, str]`right?)

Reply:
yes and done.